glib/tests/clone.rs

```diff
@@ -400,11 +400,15 @@ fn test_clone_macro_double_rename() {
     closure(0i8); // to prevent compiler error for unknown `x` type.
     let _ = clone!(@weak v, @weak w as _x => @default-return true, move || false);
 
-    let closure = clone!(@strong v as _x, @strong w => @default-return true, move |_| { false });
+    let closure = clone!(@strong v as _x, @strong w => @default-return true, move |_| {
+        false
```

Comment:
Ah does rustfmt understand the body of the `clone!` macro now because we don't go through a string?

Reply:
No idea, it was complaining about the unneeded parens and I wanted to them.